examples/timer-interrupt-rtfm.rs

```diff
@@ -54,48 +58,48 @@ const APP: () = {
 
         // Init the static resources to use them later through RTFM
         init::LateResources {
-            LED: led,
-            TIMER_HANDLER: timer,
+            led,
+            timer_handler: timer,
         }
     }
 
+    // This is optional and can be removed. When removed, RTFM automatically creates an
+    // idle loop with WFI as its body.
```

Comment:
Maybe this comment should be added instead:
https://github.com/stm32-rs/stm32f1xx-hal/pull/52/files#r270492351

Reply:
Updated the comment.